app/scripts/metamask-controller.js

```diff
@@ -302,8 +309,19 @@ export default class MetamaskController extends EventEmitter {
     this.createVaultMutex = new Mutex();
 
     this.extension.runtime.onInstalled.addListener((details) => {
-      if (details.reason === 'update' && version === '8.1.0') {
-        this.platform.openExtensionInBrowser();
+      if (details.reason === 'update') {
+        if (version === '8.1.0') {
+          this.platform.openExtensionInBrowser();
+        }
+        this.loggingController.add({
+          type: LogType.GenericLog,
+          data: {
+            event: 'Extension version update',
+            previousVersion: details.previousVersion,
```

Comment:
Should we include the current version also in case the user skips versions? I'm assuming that's possible with extensions.

Reply:
Current version is logged under `version`, do you mean changing the property name or I missed smt?

Reply:
Sorry Goktug, somehow completely missed that.